pkg/config/config.go

```diff
@@ -262,9 +262,21 @@ func LoadRemote(url string, expandEnvVars bool, c *Config) error {
 }
 
 // LoadDynamicConfiguration is used to load configuration from a variety of sources using
-// ConfigLoader, this is a templated approach
+// dynamic loader, this is a templated approach
 func LoadDynamicConfiguration(url string, _ bool, c *Config, fs *flag.FlagSet) error {
```

Comment:
nit: should expandEnv be used? (Is it possible for it to be used?)

Reply:
gomplate itself has support for environment var substitution so didn't want conflicts between the two.

Reply:
Should we return an error if expand-env is set with dynamic configs then?